charts/actions-runner-controller/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.1
+version: 0.23.2
```

```suggestion
version: 0.23.1
```

I'd appreciate it if you could leave this as-is and let me coordinate the timing of the new chart release myself! (A change in this chart version results a new chart release on merge)